src/cmake/thirdparty/SetupAxomThirdParty.cmake

```diff
@@ -234,6 +234,7 @@ if (SCR_DIR)
                         TREAT_INCLUDES_AS_SYSTEM ON
                         EXPORTABLE ON)
     blt_list_append(TO TPL_DEPS ELEMENTS scr)
+    set(SCR_FOUND ON CACHE BOOL "")
```

Comment:
CMake variable is necessary to build `spio_IO_SCR_Checkpoint` test and example: https://github.com/LLNL/axom/blob/c35431a1ee71a7c46fe54805e8aa5278120d613e/src/axom/sidre/examples/spio/CMakeLists.txt#L33-L48